test/test.coffee

```diff
@@ -69,6 +69,36 @@ describe 'livedb', ->
               throw new Error err if err
               assert.equal m.language, null
               done()
+              
+    it 'errors when oplog is missing all operations', (done) -> @create =>
+      @collection.submit @docName, {v:1, op:['test']}, (err, v) =>
+        throw new Error err if err
+        
+        getOps = @client.driver.getOps
+        @client.driver.getOps = (cName, docName, from, to, fn) ->
+          fn null, []
+        
+        @collection.submit @docName, {v:1, op:['test']}, (err, v) =>
+          assert.equal err, 'Missing operations'
+          @client.driver.getOps = getOps
+          done()
+          
+    it 'errors when oplog is missing some operations', (done) -> @create =>
```

Comment:
This is bad behaviour. If the oplog throws, that implies something has gone wrong. We shouldn't just silently eat the exception. (Exceptions ruin optimization, they shouldn't ever be used as part of the expected behaviour of a program)